client/src/components/HistoryViewer/HistoryViewer.scss

```diff
@@ -8,9 +8,19 @@
   }
 }
 
+// In compare mode, we don't want any space between the two version lists, only a small black line.
+.history-viewer__table--comparison-selected {
+  margin-bottom: 0;
+  border-bottom: 5px solid $gray-800;
+  
+  .history-viewer__row:last-child {
+    border-bottom: none;
+  }
```

Comment:
You don't need to nest this, that's one of the reasons we'd use BEM class naming

Reply:
Hmm true, it should be a modifier.